gp-includes/validation.php

```diff
@@ -226,3 +226,5 @@ public static function get( $key ) {
 GP_Validators::register( 'between', 'gp_is_between' );
 GP_Validators::register( 'between_exclusive', 'gp_is_between_exclusive' );
 GP_Validators::register( 'one_of', 'gp_is_one_of' );
+GP_Validators::register( 'consisting_only_of_ASCII_characters', 'gp_is_ascii_string' );
+GP_Validators::register( 'starting_or_ending_with_a_word_character', 'gp_is_starting_or_ending_with_a_word_character' );
```

Comment:
Rename to `starting_and_ending_with_a_word_character`  and `gp_is_starting_and_ending_with_a_word_character` to fix error message.